accounts/templates/registration/change_user_info.html

```diff
@@ -10,19 +10,26 @@
 
 {% block main_content %}
 
-<h2 class="text-center mt-1 mb-4" style="color: #e6010a;">Editing Personal Data of
-  <span class="text-dark">
-  {% if user.get_full_name %} {{ user.get_full_name }}
-  {% else %} {{ user.username }}
-  {% endif %}
-  </span>
-</h2>
-<form method="post" class="mb-3">
-
-  {% csrf_token %}
-  {% bootstrap_form form layout='horizontal' %}
-  <center>{% buttons submit='Save' %}{% endbuttons %}</center>
-
-</form>
+<div class="row justify-content-md-center">
+  <div class="col-md-auto">
+    <h2 class="text-center mt-1 mb-4" style="color: #e6010a;">Editing Personal Data of
+      <span class="text-dark">
+      {% if user.get_full_name %} {{ user.get_full_name }}
+      {% else %} {{ user.username }}
+      {% endif %}
+      </span>
+    </h2>
+    <form method="post" class="my-3">
+
```

Comment:
Alexey Radchenko said, that template tag must separate one blank line. Pair tags must separate two blank lines. Because, i everywhere did so that! Mister Dronov didn't blank lines at all. In future i will be follow your comment and recommendation.